_posts/2017/2017-08-08-why-you-should-spend-time-on-your-translation-keys.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: Why you should spend time on your translation keys
+date: "2017-08-08 09:34:02"
+image: blog-header/translation-keys.jpg
+categories: tech-stories
+authors: ["Christina"]
+---
+Translation keys are not important at all?
+Spending time on them is a waste of time?
+Let me convince you otherwise.
+
+We just finished the revision of our translation keys because we noticed the value of a good key structure.
+In the following, I will tell you why we changed the old system and what have been the most important aspects during this task.
+
+## Structure
+
+We already had functional keys before.
+Everything worked out.
+But nobody really paid attention to an overall structure or any consistency.
+For our new technology platform, we will introduce a new localization process.
+Therefore, we will work with a new tool.
+A special feature of this tool is an In-Context Editor.
```

Comment:
Hmm, the upcoming part is a bit confusing and needs to be shaped. Some thoughts:

Although a special feature of this tool is an In-Context Editor, that will already tremendously help us handling source and target texts directly in the software, there's still the need to optimize the translation keys. Because source texts are directly edited in the tool by our UX Team, for them it's hard to identify ... [explain Lenas and Anjas challenge here]